sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -67,15 +67,19 @@ def generate_batch_prefill(bs: int):
             "cs": cache_dynamic_shapes,
         }
 
+        bs_min = bs
+
         if export_config.use_extend_attention:
             bs_min = 2
             bs_max = ceildiv(llama_config.block_seq_stride, bs)
             extend_bs = torch.export.Dim("extend_bs", min=bs_min, max=bs_max)
             dynamic_shapes["tokens"][0] = extend_bs
             dynamic_shapes["seq_lens"][0] = extend_bs
             dynamic_shapes["seq_block_ids"][0] = extend_bs
-        else:
-            bs_min = bs
+        elif export_config.has_prefill_position:
+            seq_block_dim = torch.export.Dim("seq_block_dim", max=block_dim_max)
```

Comment:
I think this dimension should be called something like `tokens_seq_len_block`.

Also the above dimension
```
block_dim = torch.export.Dim("block", min=block_dim_min, max=block_dim_max)
```
Should be `seq_len_block` or `seq_len_block_id` instead of `block`. `block` is too ambiguous as we already use block to also mean a layer. It would be confusing if export fails due to some reason saying something about this dimension.